client/src/Classifier.hpp

```diff
@@ -0,0 +1,60 @@
+#ifndef classifier_h
+#define classifier_h
+
+#include <opencv2/core/core.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <opencv2/ml/ml.hpp>
+
+using namespace std;
```

Comment:
So, C++ header files are pretty dumb (basically just all get concatenated together). For that reason it's generally bad to use namespace declarations in a header since it will affect all other header files 💃